api/schemas/serializers.py

```diff
@@ -33,18 +33,13 @@ class RegistrationSchemaFormBlockSerializer(JSONAPISerializer):
 
     id = IDField(source='_id', read_only=True)
     type = TypeField()
-    page = ser.CharField(max_length=255, read_only=True)
-    section = ser.CharField(max_length=255, read_only=True)
-    help_text = ser.CharField(read_only=True, allow_blank=True)
-    block_id = ser.CharField(max_length=255)
+    answer_id = ser.CharField(max_length=255)
+    chunk_id = ser.CharField(max_length=255)
```

Comment:
`chunk_id` now links related form blocks together.  For example, a single question from the original schema, might be mapped into 5 blocks - a `question-title`, a `single-select-input` block, and three multiple choice blocks `select-input-option`, one for each choice. `chunk_id` links the relevant pieces together for easier validation.